msobooking.yaml

```diff
@@ -0,0 +1,468 @@
+entityDefinitions:
+  jailstay:
+    fqn: "ol.jailstay"
+    entitySetName: "MSOJailstay"
+    propertyDefinitions:
+      ol.id:
+        type: "ol.id"
+        column: "SYSID"
+        column: "RELDATE"
```

Comment:
why are there two columns here? Looks like the release date property got erased?

Reply:
Yes it did accidentally get erased when i was removing current class  https://github.com/Lattice-Works/Middlesex/pull/1/commits/5d30a92d4725bd4fb64fb8e9cc5190092c9efc46#diff-49d0580c380dcc7ca80cd71a81b53960L9